cmds/core/cmos/cmos.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2010-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// reads and writes to valid CMOS registers
+//
+// Description:
+//	cmos gives you an an easy way to read and write to CMOS registers
+//	using port io on x86 platforms.
+//
+// Examples:
+//	# Read register 15
+//	cmos read 15
+//	# Write to register 15
+//	cmos write 15 1
+package main
+
+import (
+	"strconv"
+	"os"
+	"log"
+	"fmt"
+	"github.com/u-root/u-root/pkg/cmos"
+	"github.com/u-root/u-root/pkg/memio"
+)
+
+var usageMsg = `usage: cmos read <index>...
+cmos write <index> <data>...
+`
+
+func usage() {
+	fmt.Print(usageMsg)
```

Comment:
nit: this should go to stderr
```suggestion
	fmt.Fprintln(os.Stderr, usageMsg)
```